src/include/daos_prop.h

```diff
@@ -445,6 +445,16 @@ daos_prop_free(daos_prop_t *prop);
 daos_prop_t *
 daos_prop_merge(daos_prop_t *old_prop, daos_prop_t *new_prop);
 
+/**
+ * Search and return a property entry of type \a type in the property list
+ * \a prop
+ * Return NULL if not found.
+ *
+ * \param[in]		prop		Property list
+ * \param[in]		type		Type of property to look for
+ */
+struct daos_prop_entry *daos_prop_entry_get(daos_prop_t *prop, uint32_t type);
+
```

Comment:
@dsikich Heads up: if/when this PR lands, I think this function `daos_prop_entry_get` could be useful for the serialization tools, since it would now be public.